docs/app/js/app.js

```diff
@@ -286,6 +286,10 @@ function(SERVICES, COMPONENTS, DEMOS, PAGES, $location, $rootScope, $http, $wind
       pages: apiDocs.service.sort(sortByName),
       type: 'toggle'
     },{
+      name: 'Types',
```

Comment:
If you don't want the types section then you could do one of a few things...

a) just bundle types and services into the same section (where everything gets it own page)
b) manually move the information about the return type into the service doc
c) build a dgeni processor that combines the service and type into the same doc